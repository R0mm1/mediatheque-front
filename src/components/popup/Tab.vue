<template>
    <div id="med_popup_tab">
        <slot name="tab_content"></slot>
    </div>
</template>

<script>
    export default {
        name: "Tab"
    }
</script>

<style scoped lang="scss">
    @import "../../assets/scss/breakpoints";

    #med_popup_tab {
        width: 100%;
        display: flex;
        overflow: auto;
        flex: 1;

        @include phone-portrait{
            flex-direction: column;
        }
    }
</style>
