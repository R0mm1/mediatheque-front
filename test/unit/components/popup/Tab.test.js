import {mount} from '@vue/test-utils'
import Tab from "@/components/popup/Tab.vue";

describe('Tab', () => {
    const wrapper = mount(Tab, {
        slots: {
            tab_content: '<div>Tab content</div>'
        }
    });

    it('renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    })
});
