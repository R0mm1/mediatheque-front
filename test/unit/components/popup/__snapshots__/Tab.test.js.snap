// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab renders correctly 1`] = `
<div
  id="med_popup_tab"
>
  <div>
    Tab content
  </div>
</div>
`;
